src/pydsl/compiler.py

```diff
@@ -364,7 +368,13 @@ def visit_Constant(self, node: ast.Constant) -> SubtreeOut:
         return Number(ast.literal_eval(node))
 
     def visit_Call(self, node: ast.Call) -> SubtreeOut:
-        return handle_CompileTimeCallable(self, node)
+        x = self.scope_stack.resolve_attr_chain(node.func)[-1]
+        if type(x).__name__ == "JITFunction":
```

Comment:
```suggestion
        if type(x).__qualname__ == "triton.JITFunction":
```
Does this also work? This should be a bit more robust.